src/utils/metrics.py

```diff
@@ -170,11 +172,18 @@ def __init__(self, labels: List[str], max_model_len: int):
 class VllmStatLogger(VllmStatLoggerBase):
     """StatLogger is used as an adapter between vLLM stats collector and Triton metrics provider."""
 
-    # local_interval not used here. It's for vLLM logs to stdout.
-    def __init__(self, labels: Dict, max_model_len: int) -> None:
+    def __init__(self, labels: Dict, max_model_len: int, logger) -> None:
```

Comment:
```suggestion
    def __init__(self, labels: Dict, max_model_len: int, log_logger) -> None:
```
Maybe rename the `logger` to `log_logger` to disambiguate between the logger that writes statistics and the logger that writes log?